delft3dworker/views.py

```diff
@@ -301,6 +301,10 @@ def post(self, request, *args, **kwargs):
                 }
             )
 
+        # hard code the tasks for the chain. This should be added to the
```

Comment:
All this is deprecated, leave the changes in: we're going to delete them anyway.